utils/manifests/definitions.go

```diff
@@ -19,6 +19,7 @@ type Component struct {
 
 type Config struct {
 	Name            string                 // Name of the service mesh,or k8 or meshery
+	Type            string                 //Type of the workload like- Istio, TraefikMesh, Kuma, OSM, Linkerd,AppMesh,NginxMesh
```

Comment:
@Revolyssup "Type" intending to serve the purpose of "registrant"?

Reply:
No this is not registerant. That will be added in adapter library

Reply:
You can go ahead and merge this

Reply:
👍